OpenRA.Renderer.Sdl2/VertexBuffer.cs

```diff
@@ -48,6 +48,16 @@ public void SetData(T[] data, int start, int length)
 			ErrorHandler.CheckGlError();
 		}
 
+		public void SetData(IntPtr data, int start, int length)
+		{
+			Bind();
```

Comment:
@WolfGaming you need separate VAOs too -- vertex format state is context state (<=2.1) or VAO state (>=3.0), not part of the buffer object.